lib/cluster/cpg.c

```diff
@@ -507,8 +507,9 @@ pcmk_message_common_cs(cpg_handle_t handle, uint32_t nodeid, uint32_t pid, void
         rc = BZ2_bzBuffToBuffDecompress(uncompressed, &new_size, msg->data, msg->compressed_size, 1, 0);
 
         if (rc != BZ_OK) {
+            rc = pcmk_bz2error2rc(rc);
```

Comment:
do the conversion before checking OK (at all calls) since multiple codes map to OK

Reply:
Fixed all call sites.  Also, there was a redundant CRM_ASSERT right after this block that can now go away.